solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -1771,6 +1787,35 @@ public void testSplitOnWhitespace_Basic() throws Exception {
     assertThat(parsedquery, anyOf(containsString("((name:stigma | title:stigma))"), containsString("((title:stigma | name:stigma))")));
   }
 
+    @Test 
+    public void testSplitOnWhitespace_stringField_shouldBuildSingleClause() throws Exception
+    {
+        assertJQ(req("qf", "trait_ss", "defType", "edismax", "q", "multi term", "sow", "false"),
```

Comment:
This is a change in behavior, and I think it's not a good change.  For a non-tokenized field (StrField in this case), I think we should ignore whatever "sow" is and split on whitespace any way, thus here have two terms to match.  It would be straight-forward to document this (no differences between numbers and StrField).

I think it could be reasonable to try both ways (both split and don't split) and then put a DisjunctionMaxQuery over the two, though I'd prefer not.